doc/source/examples/extended_examples/sfem/stochastic_fem.rst

```diff
@@ -0,0 +1,439 @@
+
+where :math:`\omega_{s,n}` is obtained as the solution of
+
+.. math:: \frac{1}{b}\cdot\tan(\omega_{s,n}\cdot a) + \omega_{s,n} = 0 \quad \text{in the range} \quad \biggl[(n-\frac{1}{2})\frac{\pi}{a}, n\frac{\pi}{a}\biggr]
+   :label: sine equation
+
+It is worth mentioning that :math:`\lambda` and :math:`\omega` in the series expansion are referred to as eigenvalue and frequency respectively.
+
+.. note::
+  In the case of an asymmetric domain, for example, :math:`\mathbb{D}=[-t_{min},t_{max}]`, a shift parameter :math:`T = (t_{min}+t_{max})/2` is required and the corresponding
+  symmetric domain becomes
+
+  .. math:: D' = D - T = \biggl[\frac{t_{min}-t_{max}}{2}, \frac{t_{max}-t_{min}}{2} \biggr]
+
+  And the series expansion becomes
+
+  .. math:: X(t) = \sum_{n=1}^\infty \sqrt{\lambda_{c,n}}\cdot\varphi_{c,n}(t-T)\cdot\xi_{c,n} + \sum_{n=1}^\infty \sqrt{\lambda_{s,n}}\cdot\varphi_{s,n}(t-T)\cdot\xi_{s,n},\quad t\in\mathbb{D}
+
+The K-L expansion of a Gaussian process has the property that :math:`\xi_{c,n}` and :math:`\xi_{s,n}` are independent
+standard normal variables, that is, they follow the :math:`\mathcal{N}(0,1)` distribution. The other great property is
+that :math:`\lambda_{c,n}` and :math:`\lambda_{s,n}` converge to zero fast (in the mean square sense). For practical implementation,
+this means that the infinite series of the K-L expansion is truncated after a finite number of terms, giving the approximation:
+
+.. math:: X(t) \approx \hat{X}(t) = \sum_{n=1}^P \sqrt{\lambda_{c,n}}\cdot\varphi_{c,n}(t)\cdot\xi_{c,n} + \sum_{n=1}^Q \sqrt{\lambda_{s,n}}\cdot\varphi_{s,n}(t)\cdot\xi_{s,n}
+   :label: approximation
+
+Equation :math:numref:`approximation` is computationally feasible to handle. A summary of how to use it to generate realizations of :math:`X(t)` follows:
+
+1. To generate the j-th realization, draw a random value for each :math:`\xi_{c,n}, n=1,\dots ,P, \quad \xi_{s,n}, n=1,\dots ,Q` from the standard
+   normal distribution :math:`\mathcal{N}(0,1)` and obtain :math:`\xi_{c,1}^j,\dots ,\xi_{c,P}^j, \quad \xi_{s,1}^j,\dots ,\xi_{s,P}^j`
+
+2. Insert these values into equation :math:numref:`approximation` in other to obtain the j-th realization:
+
+.. math:: \hat{X}^j(t) = \sum_{n=1}^P \sqrt{\lambda_{c,n}}\cdot\varphi_{c,n}(t)\cdot\xi_{c,n}^j + \sum_{n=1}^Q \sqrt{\lambda_{s,n}}\cdot\varphi_{s,n}(t)\cdot\xi_{s,n}^j
+
+3. To generate additional realizations, simply draw new random values for :math:`\xi_{c,n}, n=1,\dots ,P, \quad \xi_{s,n}, n=1,\dots ,Q` each from :math:`\mathcal{N}(0,1)`
+
+.. note::
+  In this case of a field, the preceding discussion applies as the only difference is a change in notation (for example :math:`t` to :math:`x`).
+
+The Monte Carlo simulation
+--------------------------
+For linear static problems in the context of FEM, the system equations which must be solved change from
+
+.. math:: \pmb{K}\pmb{U} = \pmb{F}
+
+to
```

Comment:
```suggestion
changes to
```
I didn't know how to change this to paragraph form easily, so I basically left it as is.